src/functions/template-tags/organizer.php

```diff
@@ -265,11 +265,25 @@ function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = fa
 
 		$org_id = tribe_get_organizer_id( $postId );
 		if ( class_exists( 'Tribe__Events__Pro__Main' ) && get_post_status( $org_id ) == 'publish' ) {
-			$url = esc_url_raw( get_permalink( $org_id ) );
+			$url    = esc_url_raw( get_permalink( $org_id ) );
+			$target = apply_filters( 'tribe_get_event_organizer_link_target', '_self' );
+			$rel    = ( '_blank' === $target  ) ? 'noopener noreferrer' : '';
+
 			if ( $full_link ) {
 				$name = tribe_get_organizer( $org_id );
-				$attr_title = the_title_attribute( array( 'post' => $org_id, 'echo' => false ) );
-				$link = ! empty( $url ) && ! empty( $name ) ? '<a href="' . esc_url( $url ) . '" title="'.$attr_title.'">' . $name . '</a>' : false;
+
+				if ( empty( $url ) || empty( $name ) ) {
+					$link = false;
+				} else {
+					$link  = sprintf(
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: a6924ce0263897116e42d47446f86988</sub>